app/static/leaderboard.css

```diff
@@ -0,0 +1,59 @@
+.rating {
+    position: relative;
+    vertical-align: middle;
+    display: inline-block;
+    color: #b1b1b1;
+    overflow: hidden;
+}
+.full-stars {
+    position: absolute;
+    left: 0;
+    top: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    color: #fde16d;
+}
+.full-peppers {
+    position: absolute;
+    left: 0;
+    top: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    color: transparent;  
+    text-shadow: 0 0 0 red;
+}
+
+.empty-stars:before, .full-stars:before {
+    content:"\2605\2605\2605\2605\2605";
+    font-size: 14pt;
+}
+.empty-stars:before {
+    -webkit-text-stroke: 1px #848484;
+}
+.full-stars:before {
+    -webkit-text-stroke: 1px orange;
+}
+
+.empty-peppers:before, .full-peppers:before {
+    content:"\01F336\01F336\01F336\01F336\01F336";
+    font-size: 14pt;
+}
+.empty-peppers:before {
+    color: transparent;  
+    text-shadow: 0 0 0 grey;
+}
+.full-peppers:before {
+    color: transparent;  
+    text-shadow: 0 0 0 red;
+}
+
+ @-moz-document url-prefix() {
```

Comment:
co robią te linijki?

Reply:
Taki problem kradzionych skryptów :D, sam musiałem rozkmniać, z tego co rozumiem: domyślnie te gwiazdki działaja na każdej przeglądarce za wyjątkiem firefox (stąd "moz" jak mozilla), w której trzeba zadbać o taki właśnie dodatek aby to smigało.